Documentation/conf.py

```diff
@@ -91,18 +91,19 @@
     chart_release = 'cilium/cilium --version ' + release
     tags.add('stable')
 relinfo = semver.parse_version_info(release)
-next_release = '%d.%d' % (relinfo.major, relinfo.minor)
+current_release = '%d.%d' % (relinfo.major, relinfo.minor)
 if relinfo.patch == 90:
     next_release = '%d.%d' % (relinfo.major, relinfo.minor + 1)
-current_release = release[0:3]
+else:
+    next_release = current_release
 githubusercontent = 'https://raw.githubusercontent.com/cilium/cilium/'
 scm_web = githubusercontent + branch
 jenkins_branch = 'https://jenkins.cilium.io/view/Cilium-v' + current_release
 archive_filename = archive_name + '.tar.gz'
 archive_link = 'https://github.com/cilium/cilium/archive/' + archive_filename
 archive_name = 'cilium-' + archive_name.strip('v')
 project_link = 'https://github.com/cilium/cilium/projects?query=is:open+' + next_release
-backport_format = 'https://github.com/cilium/cilium/pulls?q=is:open+is:pr+label:%s/' + current_release
+backport_format = 'https://github.com/cilium/cilium/pulls?q=is:open+is:pr+label:backport/%s' + current_release
```

Comment:
I _think_ this is incorrect. [Those links](https://docs.cilium.io/en/latest/contributing/release/backports/#preparation) already work. And according to [the Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/extensions/extlinks.html), the `%s` here is to be replaced in the reStructuredText document, not by `current_release`.

Reply:
Oh I missed that, thanks for checking! I'll fix.